codespell_lib/data/dictionary.txt

```diff
@@ -12772,6 +12772,7 @@ cirularly->circularly
 cirumflex->circumflex
 cirumstance->circumstance
 cirumstances->circumstances
+cirumvent->circumvent
```

Comment:
Thank you for this contribution. It would be great if you could add all verbal forms:
```suggestion
cirumvent->circumvent
cirumvented->circumvented
cirumventing->circumventing
cirumvents->circumvents
```

Reply:
Thanks, was unable to apply from here, just did locally and added an extra noun form.